modules/renter/renter.go

```diff
@@ -491,13 +491,11 @@ func NewCustomRenter(g modules.Gateway, cs modules.ConsensusSet, tpool modules.T
 	}
 	r.memoryManager = newMemoryManager(defaultMemory, r.tg.StopChan())
 
-	// Load all saved data.
-	id := r.mu.Lock()
+	// Load all saved data. We are still in the renter's constructor so it
+	// should be fine to call an unexported method without holding the lock.
```

Comment:
Well it's not completely wrong. Even without any goroutines spawned by the renter it might be accessed from multiple threads via the API :P 
Either way I removed the comment because it seems unnecessary since we don't explain that in any other module.

Reply:
No because during the constructor stage the renter won't have been returned to anyone yet, it's not possible for API calls to be interfering with it.

So I guess the full comment would say `we're in the constructor and also we haven't created any goroutines yet`